eng/scripts/Language-Settings.ps1

```diff
@@ -132,7 +132,7 @@ function Get-rust-AdditionalValidationPackagesFromPackageSet ($packagesWithChang
   [array]$additionalPackages = $affectedPackages | Where-Object { $packagesWithChanges -notcontains $_ }
 
   # if the change affected no packages, e.g. eng/common change, we use core and template for validation
-  if ($additionalPackages.Length -eq 0) {
+  if ($packagesWithChanges.Length -eq 0) {
```

Comment:
```suggestion
  if ($additionalPackages.Length -eq 0 -and $packagesWithChanges.Length -eq 0) {
```

Reply:
We're wanting to express:
- $packagesWithChanges will already contain packages where $diff contains a file in the package directory
- $additionalPackages will contain any packages where there was a change in its service directory, but not in a package directory.  e.g.     /sdk/cosmos/some-file.json
- If we have no auto-discovered packages, use _azure_core and azure_template